app/pages/users/family/family_list.php

```diff
@@ -18,4 +18,5 @@
     <?php endforeach;
     if (!$families): ?>
         <p class="center">Pas encore de familles 🥺</p>
-    <?php endif ?>
\ No newline at end of file
+    <?php endif ?>
+</table>
```

Comment:
lmao

Reply:
Actually it might be nice to move this condition on top of the rendering and early return